wsidicomizer/openslide.py

```diff
@@ -79,6 +107,19 @@ def __init__(
         """
         super().__init__(encoder)
         self._slide = open_slide
+        slide_background_color_string = self._slide.properties[
+            PROPERTY_NAME_BACKGROUND_COLOR
+        ]
+        if slide_background_color_string is None:
+            self._blank_color = super()._get_blank_color(
+                self.photometric_interpretation
+            )
+        else:
+            self._blank_color = (
+                int(slide_background_color_string[0:2], 16),
```

Comment:
This looks like quite opportunistic parsing, would it be worth validating that the slide_background_color_string is on the appropriate format using a regex before parsing it like this?

I guess the string could have any format for all you know?

Reply:
True. Changed to a simple parser to get 3 hex bytes.